.github/workflows/helm.yaml

```
@@ -0,0 +1,42 @@
+---
+name: Helm Chart Release Action
+on:
+  pull_request:
+    branches:
+        - main
+  release:
+    types:
+        - published
+  push:
+    branches:
+        - main
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    name: Docker Build
+    runs-on: ubuntu-latest
+
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+
+        - name: Make tag
+          run: |
+            [ "${GITHUB_EVENT_NAME}" == 'pull_request' ] && echo "tag=v0.0.0-pull-request.${{ github.event.number }}" >> $GITHUB_ENV || true
+            [ "${GITHUB_EVENT_NAME}" == 'release' ] && echo "tag=${GITHUB_REF##*/}" >> $GITHUB_ENV || true
+            [ "${GITHUB_EVENT_NAME}" == 'push' ] && echo "tag=v0.0.0" >> $GITHUB_ENV || true
```

Looks strange without any suffix.
```suggestion
            [ "${GITHUB_EVENT_NAME}" == 'push' ] && echo "tag=v0.0.0-develop" >> $GITHUB_ENV || true
```